ccmbioinfo/lfs
# CARDS-1599: Subject resources can be serialized in Text (.txt) and Markdown (.md) formats

### Specifications:
[CARDS-1599](https://phenotips.atlassian.net/browse/CARDS-1599)

### Testing instructions:
* Create a subject
* Add some forms with answers to it
* navigate to `localhost:8080/Subjects/FULL/SUBJECT/ID.txt` to see the text serialization
* navigate to `localhost:8080/Subjects/FULL/SUBJECT/ID.md` to see the markdown serialization
* To ensure there are no regressions, also test individual form serialization by navigating to:
`localhost:8080/Forms/FORM-ID.txt` to see the text serialization
`localhost:8080/Forms/FORM-ID.md` to see the markdown serialization
